src/simulator.zig

```diff
@@ -143,9 +143,10 @@ pub fn main() !void {
     const simulator_options = Simulator.Options{
         .cluster = cluster_options,
         .workload = workload_options,
+        // TODO Swarm testing: Test long+few crashes and short+many crashes separately.
         .replica_crash_probability = 0.00002,
         .replica_crash_stability = random.uintLessThan(u32, 1_000),
-        .replica_restart_probability = 0.0001,
+        .replica_restart_probability = 0.0002,
```

Comment:
Tuning this a little since I ran into some simulator runs that took a very long time simply because they were blocked waiting for a particular replica to restart. Once state transfer is far enough along that the "whole quorum must checkpoint together" hack is removed, we'll be able to test much longer crashes.